### ISSUE TITLE
savehistory() does not work in Radian sessions

### ISSUE THREAD

**Issue by vtdmhd** [randy3k/radian#98]
In an interactive REPL session, dumping the current history can be quite useful.
However, 
`r$> savehistory(file='/tmp/asd')`
`Error in .External2(C_savehistory, file) : no history available to save`

If Radian replaces this with some more sophisticated mechanism, probably some hint message at this point would be useful.

**Comment by randy3k** [comment]
You are right, the saving function won’t work in radian. radian handles history differently and uses its own history file. The message is produced by r internal code. I am not sure if there’s a way to control the message.

**Comment by nk027** [comment]
I bypassed the error message by adjusting my `.Rprofile`:
```
 .Last <- function() {
   hist_file <- Sys.getenv("R_HISTFILE")
   if(hist_file == "") hist_file <- "~/.Rhistory"
   # Radian handles history differently
   if(interactive()) tryCatch(utils::savehistory(hist_file),
     error = function(e) {message("Could not save history.")})
 }
```

**Comment by randy3k** [comment]
@nk027 You might use `nzchar(Sys.getenv("RADIAN_VERSION"))` to check if it is radian.

**Comment by vtdmhd** [comment]
@randy3k @nk027 This doesn't make savehistory() work

**Comment by eantonya** [comment]
`history()` function doesn't work either for this same reason.

**Comment by randy3k** [comment]
@eantonya why is it needed in the first place? It should be either `~/.radian_history` or any local `.radian_hisotry`. Also depends on how you start `radian`. For example, `radian --global-history` will also use the history in the home directory.

**Comment by eantonya** [comment]
@randy3k my workflow often involves pulling up history, seeing what sequence of commands I've run, copy-pasting multiple lines from it, etc. If native `history()` command is not supported, I can write a workaround myself if I know what file to read.

**Comment by tongtong1993** [comment]
@randy3k It is really good to have something similar to history(). As I normally build pipeline using the interactive console, after completing some of the code, I will use history() to copy and paste to the text editor.
